### PR TITLE
Feature/modernize

### PR DESCRIPTION
**Pull request by Chris53897**
Hi, i did some modernizations.

- drop support of php < 7.4 (EOL)
- fix github actions cs2pr
- drop support of symfony 3 (EOL)
- allow symfony 6
- update phpunit 9
- remove deprected psaml option `totallyTyped="true"`
- composer.json allow plugins (as suggested by composer. Otherwise will not work after 22.05.2022 if i read correct)

WDYT?

### CODE REVIEW

**Comment by Chris53897** [comment]
@malukenho Hi. Can you have a look at this PR?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,12 +9,18 @@ jobs:
 
     strategy:
       matrix:
-        php: ['7.1', '7.2', '7.3', '7.4', '8.0']
```
Comment (malukenho) [inline .github/workflows/main.yml:12]: May we drop support for PHP 7.4? The [EOL](https://www.php.net/supported-versions.php) is nearby.

**Review by malukenho — CHANGES_REQUESTED** [review]

**Comment by malukenho** [comment]
Other than the PHP-supported version looks fine.

File: .github/workflows/main.yml (hunk 1 above)
Comment (Chris53897) [reply .github/workflows/main.yml:12]: Yes. I did drop support for 7.4, and add features of php8

**Comment by malukenho** [comment]
I'll tag a version tonight.

**Comment by Chris53897** [comment]
@malukenho Can you please tag it?

**Comment by malukenho** [comment]
@Chris53897 https://github.com/malukenho/docheader/releases/tag/1.0.0

**Pull request merged**